sound/soc/sof/ipc4-topology.c

```diff
@@ -232,6 +232,38 @@ static int sof_ipc4_setup_process_init_byte_kcontrol(struct snd_sof_widget *swid
 	return 0;
 }
 
+static const struct sof_ipc4_audio_format
```

Comment:
We should use the terms "built-in modules" for modules that are in the base FW and external modules for those that are not, loadable doesnt quite seem right.

Reply:
globally in this PR use built-in/external module instead of loadable